src/System.Management.Automation/engine/parser/PSType.cs

```diff
@@ -265,6 +265,39 @@ internal static void DefineCustomAttributes(EnumBuilder member, ReadOnlyCollecti
             }
```

Comment:
@IISResetMe, your last commit had 4 failures in `PowerShell-CI-macos`
Error occurred in test script '/Users/runner/runners/2.169.1/work/1/s/test/powershell/Language/Classes/scripting.Classes.inheritance.tests.ps1'
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, /Users/runner/runners/2.169.1/work/1/a/bins/publish/Modules/Pester/4.10.1/Pester.psm1: line 1111
```
Error occurred in test script '/Users/runner/runners/2.169.1/work/1/s/test/powershell/Language/Classes/scripting.Classes.inheritance.tests.ps1'
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, /Users/runner/runners/2.169.1/work/1/a/bins/publish/Modules/Pester/4.10.1/Pester.psm1: line 1111
```
Error occurred in test script '/Users/runner/runners/2.169.1/work/1/s/test/powershell/Language/Classes/scripting.Classes.inheritance.tests.ps1'
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, /Users/runner/runners/2.169.1/work/1/a/bins/publish/Modules/Pester/4.10.1/Pester.psm1: line 1111
```
Error occurred in test script '/Users/runner/runners/2.169.1/work/1/s/test/powershell/Language/Classes/scripting.Classes.inheritance.tests.ps1'
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, /Users/runner/runners/2.169.1/work/1/a/bins/publish/Modules/Pester/4.10.1/Pester.psm1: line 1111
```